test/e2e/crcsuite/crcsuite.go

```diff
@@ -23,13 +23,13 @@ import (
 )
 
 var (
-	CRCHome        string
-	CRCExecutable  string
-	bundleEmbedded bool
-	bundleName     string
-	bundleLocation string
-	pullSecretFile string
-	cleanupHome    bool
+	CRCHome                   string
+	CRCExecutable             string
+	bundleDownloadedAutomatic bool
```

Comment:
Not sure about the name of this variable. `bundleDownloadAutomatic`, or `defaultBundle` or inverting true/false values with `userProvidedBundle` might be more intuitive.